src/k8s-job-exit-spec/config/k8s-job-exit-spec.yaml

```diff
@@ -926,21 +926,24 @@ spec:
     - userLogRegex: "(?i)cuda runtime error (2) : out of memory"
 
 - code: 227
-  phrase: ContainerEngineCoreDump
+  phrase: ContainerGpuDeviceEccError
   issuer: PAI_RUNTIME
-  causer: USER_CONTAINER
-  type: USER_FAILURE
+  causer: PAI_HW
+  type: PLATFORM_FAILURE
   stage: RUNNING
-  behavior: PERMANENT
-  reaction: NEVER_RETRY
-  reason: "Container failed due to engine core dump"
+  behavior: TRANSIENT
+  reaction: RETRY_TO_MAX
+  reason: "Container failed due GPU ecc error"
   repro:
-  - "Access invalid memory address"
+  - "Initialize tensorflow streamExecutor in GPU with ECC error"
   solution:
-  - "Correct the user code"
+  - "Wait result from next retry"
+  - "Contact Cluster Admin"
   pattern:
     runtimeContainerPatterns:
-    - userLogRegex: "(?i)core dumped"
+    - userLogRegex: "(?i)CUDA_ERROR_ECC_UNCORRECTABLE: uncorrectable ECC error encountered"
+    - gpuInfo:
```

Comment:
Do we still need ContainerCudaUncorrectableECCError?

Reply:
Thanks, I found there already a pattern to match `CUDA_ERROR_ECC_UNCORRECTABLE`
```yaml

- code: 223
  phrase: ContainerCudaUncorrectableECCError
  issuer: PAI_RUNTIME
  causer: PAI_HW
  type: PLATFORM_FAILURE
  stage: RUNNING
  behavior: TRANSIENT
  reaction: ALWAYS_RETRY
  reason: "Container failed due to GPU uncorrectable ECC error"
  repro:
  - "Run tensorflow on a GPU which has ECC error"
  solution:
  - "Wait result from next retry"
  - "Contact Cluster Admin"
  pattern:
    runtimeContainerPatterns:
    - userLogRegex: "(?msi)CUDA_ERROR_ECC_UNCORRECTABLE.*"
```
Change name to `ContainerMayFailDueToGpuDeviceEccError` and remove `userLogRegex`

In some case ecc error with just cause segment fault/core dump without any log